earn/src/components/markets/modal/SupplyModal.tsx

```diff
@@ -67,6 +76,8 @@ function getConfirmButton(state: ConfirmButtonState, token: Token): { text: stri
       return { text: 'Check Wallet', enabled: false };
     case ConfirmButtonState.READY:
       return { text: 'Confirm', enabled: true };
+    case ConfirmButtonState.WAITING_FOR_CONFIRMATION:
+      return { text: 'Please Confirm', enabled: false };
```

Comment:
Seems unused? Also, I believe we say "Check wallet" as the button text for this type of thing in other places.

Reply:
Yep, left over from an earlier implementation that required confirming explicitly